templates/vue/src/components/Lightbox/index.vue

```diff
@@ -82,10 +87,11 @@ export default {
       },
       showCompletionScreen: false,
       rowRefs: [],
+      selectedTab: "",
```

Comment:
We don't seem to be using this in this component anywhere?

Reply:
Currently it is not used but it is relevant for tyde activities because we want to alternate between normal node and tyde views such as cos